UQ-eLIPSE/crucible-search
# Feature/vethub 240

update style and replace div with input element:
to test:
- `yarn test`
- to verify, run `yarn dev` on your local, should see:
![image](https://github.com/user-attachments/assets/66bfd530-3a26-4507-a3db-f94e9ace7bb3)
